examples/src/App.tsx

```diff
@@ -46,18 +48,36 @@ const CustomTabPanel = ({
 
 function App() {
   const { chain } = useNetwork();
+  const { chains, switchNetwork } = useSwitchNetwork();
+
+  const [sdk, setSdk] = useState(new EndaomentSdkApi(new Configuration({ network: 'goerli' })));
+
+  const handleChainChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    const chainId = +e.target.value;
+    switchNetwork?.(chainId);
+
+    setSdk(new EndaomentSdkApi(new Configuration({ network: chainId === 1 ? 'mainnet' : 'goerli' })));
+  };
 
   return (
     <Container maxW="4xl" p="4" mt="16">
       <Flex justifyContent="space-between" alignItems="center" mb={8}>
         <Heading size="md">Endaoment SDK - Examples</Heading>
         <VStack spacing="1">
           <ConnectKitButton theme="soft" />
-          {chain && (
-            <Text fontSize="xs">
-              Chain: {chain.name} ({chain.id})
-            </Text>
-          )}
+
+          <Select
+            variant="flushed"
+            value={chain?.id}
+            onChange={handleChainChange}
+            placeholder={chain?.id ? 'Select Network' : '👆 Waiting for Connect'}
```

Comment:
Why not make it conditionally display only if the user is connected?

Reply:
That would make too much sense for my taste